hvplot/tests/testutil.py

```diff
@@ -283,20 +291,58 @@ def test_check_crs():
 
 @pytest.mark.parametrize("input", [
     "+init=epsg:26911",
+], ids=lambda x: str(x)[:20])
```

Comment:
```suggestion
])
```
NIT: The only reason I added ids, was not to have a big string in the CI for the long proj definition